src/html/elevations/elevations.html

```diff
@@ -1,4 +1,3 @@
-<div class="ds-elevation-remove"></div>
```

Comment:
@pmmck We currently don't have a class called ds-elevation-remove in the DS. This is being used only as a mixin currently. Do we want to create that class?